src/main/java/com/tastetracker/web/api/ReviewRestController.java

```diff
@@ -55,11 +48,16 @@ public ResponseEntity<RatingDto> getRating(@PathVariable long restaurantId){
     @GetMapping("/checkReviewStatus/{restaurantId}")
     public ResponseEntity<Boolean> checkReviewStatus( @PathVariable long restaurantId)
     {
-        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
-        String login = authentication.getName();
+        String login = getLoginFromContext();
         UserDto user = userService.findUserByLogin( login )
             .orElseThrow( () -> new ResponseStatusException( HttpStatus.NOT_FOUND ) );
         boolean hasUserReviews = reviewService.hasUserReviewsRestaurant( user.id(), restaurantId );
         return ResponseEntity.ok(hasUserReviews);
     }
+
+    private static String getLoginFromContext()
```

Comment:
Potrzebna tu jest na to osobna metoda? SecurityContextHolder.getContext() jest statyczne, zapisałbym to po prostu 

String login = SecurityContextHolder.getContext().getAuthentication().getName();